### DISCUSSION TITLE
Commits from VS Code are committed under different name/account

### DISCUSSION THREAD

**Discussion by RaghavT5** [community/community#45109] [New to GitHub]
As far as I know I am signed into the same account on VS Code as on Chrome. In fact its my only account. But for some reason, all my commits that I push from VS Code have a different name linked to it. I mean, it shows my name but it's not linked to my account. You can't click on it to see my account. Commits I make through VSC are not counted or reflect in my account/commit activity. I am not sure what the issue is.

Edit: The issue is limited to VSC. Things work perfectly fine when am using Android Studio.

**Comment by jge162** [comment]
You need to make sure everything is configured, here are some things to review @RaghavT5 

```git config --global user.name "Your Name" 
git config --global user.email "your_email@example.com"
Replace "Your Name" and "[your_email@example.com](mailto:your_email@example.com)" with your name and email address.

Also please check if you are using a different ssh key for vs code.
You can check your ssh key by running

cat ~/.ssh/id_rsa.pub```

Also, make sure you have added your ssh key to your account on chrome.

**Comment by meesam4687** [comment]
Like the person before said, you need to configure git first.
Along with email you might need to configure the name as well
```
git config --global user.name <YOUR_GITHUB_USERNAME>
git config --global user.email <YOUR_EMAIL>
```

**Reply by KayG2310** [reply]
Thank you, this was helpful!
